warehouse/models/mart/transit_database/dim_contract_attachments.sql

```diff
@@ -1,18 +1,22 @@
 {{ config(materialized='table') }}
 
-WITH stg_transit_database__contracts AS (
-    SELECT * FROM {{ ref('stg_transit_database__contracts') }}
+WITH latest AS (
```

Comment:
nit: I think aliasing all of these latest CTEs with `latest` is a tad confusing but don't have to change now

I think something like `latest_contracts` would be clearer?

Reply:
✅ https://github.com/cal-itp/data-infra/pull/1625/commits/d157f6f46ccd0be1ae446e6daf46a5887da79a66